ysh/funcs_builtin.py

```diff
@@ -192,6 +195,39 @@ def Init3(mem, config_parser, eval_to_dict, block_as_str, hay_result):
     SetGlobalFunc(mem, 'block_as_str', block_as_str.Call)
 
 
+class _Bool(vm._Func):
+
+    def __init__(self):
+        # type: () -> None
+        vm._Func.__init__(self)
+
+    def Run(self, pos_args, named_args):
+        # type: (List[value_t], Dict[str, value_t]) -> value_t
+        assert len(pos_args) == 1
+        assert len(named_args) == 0
+        return value.Bool(expr_eval.ToBool(pos_args[0]))
```

Comment:
Yeah actually this is a case where we need an internal ToBool() to implement `and` and `or` operators

But then we ALSO have a wrapper Bool(vm._Func)  to expose `Bool()` to the user

So it makes sense